internal/indexer/indexer.go

```diff
@@ -149,10 +156,17 @@ func (i *Indexer) loadPackages(deduplicate bool) error {
 			Logf:  i.packagesLoadLogger,
 		}
 
-		pkgs, err := packages.Load(config, "./...")
-		if err != nil {
-			errs <- errors.Wrap(err, "packages.Load")
-			return
+		// Make sure we only load packages once per execution.
+		pkgs, ok := cachedPackages[i.projectRoot]
```

Comment:
@tjdevries is this was what you referring to as future-proofing other calls to `packages.Load`?

Reply:
Yeah, that's the one.